src/ankisyncd/sync_app.py

```diff
@@ -35,24 +35,25 @@
 from webob.exc import *
 
 import anki.db
-import anki.sync
 import anki.utils
-from anki.consts import SYNC_VER, SYNC_ZIP_SIZE, SYNC_ZIP_COUNT
 from anki.consts import REM_CARD, REM_NOTE
 
 from ankisyncd.users import get_user_manager
 from ankisyncd.sessions import get_session_manager
 from ankisyncd.full_sync import get_full_sync_manager
 
+from .sync import Syncer, SYNC_VER, SYNC_ZIP_SIZE, SYNC_ZIP_COUNT
```

Comment:
Can you use an absolute import: `from ankisyncd import ...`

Reply:
Agreed, I think both are fine but like 'tabs or spaces?', the correct answer is 'whatever is there when you started, unless you are prepared to convince all current contributors and migrate everything, personally supporting any and all issues that arise' :-).

Reply:
Yes, fixed it.